jest.config.ts

```diff
@@ -45,13 +45,13 @@ export default {
   projects: [
     {
       ...baseConfig,
-      displayName: "browser [Timezone: UTC]",
```

Comment:
Why the name change (change is also unrelated to the main contents of this PR).

Reply:
https://github.com/inrupt/solid-client-js/blob/a3558586109e2374653121e1073db97a72f16a61/package.json#L21 resulted in `npm run test:unit:browser` failing, because there was no project matching the `browser` name. This went unnoticed previously, because `npm run test` ran all unit test suites by simply excluding the e2e ones, but the change to `.github/workflows/ci.yml` resulted in each test suite running independantly. That was causing CI to fail.